pages/web/web-paas/administration-users/guide.en-asia.md

```diff
@@ -55,24 +56,30 @@ The following table shows the available roles for environment types.
 | Contributor | Yes | Yes | Yes | Yes | No | No |
 | Admin| Yes | Yes | Yes | Yes | Yes | Yes |
 
-To customize who can use SSH, [set the access key](../configuration-app/access/) in your `platform.app.yaml` file.
+To customize who can use SSH, [set the access key](../configuration-app/access) in your `platform.app.yaml` file.
 
 ## Manage users
 
 ### Add a user to a project
 
-If you are an organization owner, or an organization user with manage plan or manage users permissions wanting to add a user to a project or an environment, follow these steps:
+To add a user, you need to be a [Project Admin](#user-roles).
+
+To add a user to a project or an environment, follow these steps:
 
 > [!tabs]      
 > In the console     
 >> ``` false     
 >> 
 >> 
->> 1. In the console, select the project where you want to add a new user.
->> 2. Under **Settings**, click **Access**.
->> 3. Click **+ Add**.
->> 4. Add the user's details and choose their permissions.
->> 5. Click **Save**.
+>> <!--This is in HTML to get the icon not to break the list. -->
+>> <ol>
+>>   <li>Select the project where you want to add a new user.</li>
+>>   <li>Click {{< icon settings >}} <strong>Settings</strong>.</li>
```

Comment:
Hello @sreedhards, this formatting {{< icon settings >}} won't be rendered when deployed. Could you please remove it ?

Reply:
Updated